src/MatrixAlgebraKit.jl

```diff
@@ -3,16 +3,16 @@ module MatrixAlgebraKit
 using LinearAlgebra: LinearAlgebra
 using LinearAlgebra: norm # TODO: eleminate if we use VectorInterface.jl?
 using LinearAlgebra: mul!, rmul!, lmul!, adjoint!, rdiv!, ldiv!
-using LinearAlgebra: sylvester
-using LinearAlgebra: isposdef, issymmetric
+using LinearAlgebra: sylvester, lu!
+using LinearAlgebra: isposdef, issymmetric, opnorm
 using LinearAlgebra: Diagonal, diag, diagind, isdiag
 using LinearAlgebra: UpperTriangular, LowerTriangular
 using LinearAlgebra: BlasFloat, BlasReal, BlasComplex, BlasInt
 
 export isisometry, isunitary, ishermitian, isantihermitian
 
-export project_hermitian, project_antihermitian
-export project_hermitian!, project_antihermitian!
+export project_hermitian, project_antihermitian, project_isometric
+export project_hermitian!, project_antihermitian!, project_isometric!
```

Comment:
I'm slightly unhappy with `project_isometric` vs `isisometry`. Is it at all reasonable to have `project_isometry`, or does that not really make sense? Happy to leave as is too, just noticing this here.

Reply:
That is a good catch. But I would prefer `isisometric`, as all other checks are also using the adjective form, e.g. `ishermitian`, ...

I know this constitutes yet another breaking change 😄 .

I guess unitary is the odd duck, as that is both an adjective and a noun.